<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmLunarCalendarWidget</class>
 <widget class="QWidget" name="frmLunarCalendarWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/image/peking_university.png);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="LunarCalendarWidget" name="lunarCalendarWidget" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">#lunarCalendarWidget *{background-image: url(:/image/peking_university.png);}</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labCalendarStyle">
       <property name="text">
        <string>整体样式</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cboxCalendarStyle">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>红色风格</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labSelectType">
       <property name="text">
        <string>选中样式</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cboxSelectType">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>矩形背景</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>圆形背景</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>角标背景</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>图片背景</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labWeekNameFormat">
       <property name="text">
        <string>星期格式</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cboxWeekNameFormat">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>短名称</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>普通名称</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>长名称</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>英文名称</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ckShowLunar">
       <property name="text">
        <string>显示农历</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QTextEdit" name="textEdit">
     <property name="styleSheet">
      <string notr="true">#textEdit *{background-image: url(:/image/peking_university.png);}</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>从文件中读取</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>保存为文本</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>textEdit</zorder>
  <zorder>lunarCalendarWidget</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LunarCalendarWidget</class>
   <extends>QWidget</extends>
   <header>lunarcalendarwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
